pkgs/tools/misc/uwufetch/default.nix

```diff
@@ -1,20 +1,25 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, makeWrapper, viu }:
+{ lib, stdenv, fetchFromGitHub, makeWrapper, viu }:
 
 stdenv.mkDerivation rec {
   pname = "uwufetch";
-  version = "2.0";
+  version = "2.1";
 
   src = fetchFromGitHub {
     owner = "TheDarkBug";
     repo = pname;
     rev = version;
-    hash = "sha256-2kktKdQ1xjQRIQR2auwveHgNWGaX1jdJsdlgWrH6l2g=";
+    hash = "sha256-cA8sajh+puswyKikr0Jp9ei+EpVkH+vhEp+pTerkUqA=";
   };
 
-  patches = [
-    # cannot find images in /usr
-    ./fix-paths.patch
-  ];
+  postPatch = ''
+    substituteInPlace uwufetch.c \
+      --replace "/usr/lib/uwufetch" "$out/lib/uwufetch" \
+      --replace "/usr/local/lib/uwufetch" "$out/lib/uwufetch" \
+      --replace "/etc/uwufetch/config" "$out/etc/uwufetch/config"
+    # fix command_path for package manager (nix-store)
+    substituteInPlace fetch.c \
+      --replace "/usr/bin" "/run/current-system/sw/bin"
```

Comment:
for darwin, the `PKGPATH` is `"/usr/local/bin"`. Do we know what the proper substitute would be?

Reply:
`/usr/local/bin/brew`,  I think it's fine to keep this path